Source/WebCore/rendering/RenderBlockFlow.cpp

```diff
@@ -1022,7 +1024,8 @@ void RenderBlockFlow::layoutBlockChild(RenderBox& child, MarginInfo& marginInfo,
     if (childNeededLayout)
         child.layout();
 
-    if (auto blockStepSizeForChild = child.style().blockStepSize())
+    auto& childStyle = child.style();
+    if (auto blockStepSizeForChild = childStyle.blockStepSize(); blockStepSizeForChild && BlockStepSizing::childHasSupportedStyle(child.style()))
```

Comment:
why child.style() again if you took a reference earlier  which you also in that line ?